lib/ansible/modules/cloud/azure/azure_rm_cosmosdbaccount_facts.py

```diff
@@ -125,7 +125,7 @@
                     sample: 100
         failover_policies:
             description:
-                - Read locations
+                - Read locations.
```

Comment:
This doesn't seem right. Shouldn't this be "Failover policies" or something like that?